### DISCUSSION TITLE
Auto-load nvim-tree and hide dotfiles on startup

### DISCUSSION THREAD

**Discussion by TahsanBinAlam** [NvChad/NvChad#1934] [Q&A] [answered]
Hey,
So every time I open nvim I have to open up the nvim-tree and press H to hide the dotfiles (as I don't like clutter). It kinda feels cumbersome to do that everytime I open nvim. How can i set this as default behavior?

**Comment by siduck** [comment, marked as answer]
hi @TahsanBinAlam the dotfiles are hidden by default i think. 

and to open nvimtree by default add this in your custom init.lua







```lua
vim.schedule(function()
  require("nvim-tree.api").tree.open()
end)
```
